data-raw/healthy-places/living-conditions/rough-sleeping.R

```diff
@@ -0,0 +1,80 @@
+# ---- Installing  libraries ----
+library(tidyverse)
+library(readxl)
+library(geographr)
+
+# ---- Upload csv files for rough sleepers and population data ----
+# Source: https://statswales.gov.wales/Catalogue/Housing/Homelessness/homelessness-accommodation-provision-and-rough-sleeping/roughsleepers-by-localauthority. 
+# Data needs to be manually downloaded and saved in the raw-data/ folder
+rough_sleepers_data <- read.csv("~/GitHub/health-index-wales/data-raw/healthy-places/living-conditions/raw-data/rough_sleepers_data.csv")
+
+# Source: https://statswales.gov.wales/Catalogue/Population-and-Migration/Population/Estimates/Local-Authority/populationestimates-by-localauthority-year.
+# Data needs to be manually downloaded and saved in the raw-data/ folder
+population_df_wales <- read.csv("~/GitHub/health-index-wales/data-raw/healthy-places/living-conditions/raw-data/export.csv")
+
+# Filter to include mid-year 2023 data
+population_df_wales <- population_df_wales |>
+  select(`X.2`, `X.11`) |>
+  slice(12:33)
+
+# Filter to include data from June 2023 - June 2024 
+rough_sleepers_data <- rough_sleepers_data |>
+  select(1, `X.2`:`X.14`) |>
+  slice(4:25) 
+
+# ---- Merge data sets to  calculate the % of people in each LA who are rough ----
+# sleepers, and clean
+# Rename first column in each data frame to local_authority ---
+names(rough_sleepers_data)[1] <- "local_authority"
+names(population_df_wales)[1] <- "local_authority"
+
+rough_sleepers_data <- rough_sleepers_data |>
+  mutate(across(`X.2`:`X.14`, as.numeric))
+
+rough_sleepers_data <- rough_sleepers_data |>
+  rowwise() |>
+  mutate(year_sum = sum(c_across(`X.2`:`X.14`), na.rm = TRUE)) |>
+  ungroup() # sum for number of rough sleepers per LA June 2023 to June 2024
+
+population_df_wales <- population_df_wales |>
+  mutate(across(`X.11`, as.numeric))
+
+hp_rough_sleepers <- rough_sleepers_data |>
+  left_join(population_df_wales, by = c("local_authority"))
+
+# Remove monthly columns from June-2023 to June 2024
+hp_rough_sleepers <- hp_rough_sleepers[, -2:-14]
+
+names(hp_rough_sleepers)[3] <- "pop_av"
```

Comment:
Please use tidyverse syntax

Is pop_av the most descriptive name for what this col represents?